backend/cfg/cfg_with_layout.ml

```diff
@@ -98,18 +86,16 @@ let remove_blocks t labels_to_remove =
     (* CR-soon xclerc: would be simpler with a function such as
        `DoublyLinkedList.remove : 'a cell -> unit` called from
        `DoublyLinkedList.iter_cell` *)
```

Comment:
(That function now exists, as `delete_curr`.)

Reply:
fixed